sdcm/argus_results.py

```diff
@@ -32,6 +32,126 @@
     }
 }
 
+DEFAULT_VALIDATION_RULES = {
+    "P90 write": ValidationRule(fixed_limit=5),
+    "P90 read": ValidationRule(fixed_limit=5),
+    "P99 write": ValidationRule(fixed_limit=10),
+    "P99 read": ValidationRule(fixed_limit=10),
+    # will fail sometimes: https://github.com/scylladb/scylla-cluster-tests/issues/9294
+    "Throughput write": ValidationRule(best_abs=2000),
+    "Throughput read": ValidationRule(best_abs=2000),
+}
+
+WORKLOAD_VALIDATION_RULES = {
```

Comment:
can we move this definitions to SctConfiguration ?

so we can set those differently per test if needed ?

alos the steps are defined by configuration, so it can be easily missing, and we'll have one without validation rules, and we won't know...

Reply:
sure, can be moved to sct_config, main question then if we proceed with current way: <workload>.<name>.metric and if not provided using defaults

Reply:
I'm o.k. for a small set of defaults

Like P95 and P99, the rest are really specific